bosh/releases/pre_render_scripts/scheduler/cloud_controller_clock/jobs/patch-cc.sh

```diff
@@ -17,144 +17,36 @@ patch --verbose "${target}" <<'EOT'
 
  source /var/vcap/jobs/cloud_controller_clock/bin/ruby_version.sh
  cd /var/vcap/packages/cloud_controller_ng/cloud_controller_ng
-+patch -p0 < /var/vcap/all-releases/jobs-src/capi/cloud_controller_ng/yaml-anchor.patch
++patch -p0 < /var/vcap/all-releases/jobs-src/capi/cloud_controller_ng/opi-task.patch
 +
  exec bundle exec rake clock:start
 EOT
 
 sha256sum "${target}" > "${sentinel}"
 
-cat <<'EOT' > /var/vcap/all-releases/jobs-src/capi/cloud_controller_ng/yaml-anchor.patch
-diff --git app/controllers/v3/app_manifests_controller.rb app/controllers/v3/app_manifests_controller.rb
-index ed3ca78a6..9828cd7ed 100644
---- app/controllers/v3/app_manifests_controller.rb
-+++ app/controllers/v3/app_manifests_controller.rb
-@@ -70,7 +70,7 @@ class AppManifestsController < ApplicationController
-   def validate_content_type!
-     if !request_content_type_is_yaml?
-       logger.error("Context-type isn't yaml: #{request.content_type}")
--      invalid_request!('Content-Type must be yaml')
-+      bad_request!('Content-Type must be yaml')
+cat <<'EOT' > /var/vcap/all-releases/jobs-src/capi/cloud_controller_ng/opi-task.patch
+diff --git lib/cloud_controller/opi/task_client.rb lib/cloud_controller/opi/task_client.rb
```

Comment:
It would be nice if we can get a git commit message or some comments describing why we want this patch. (I understand that it's because our Eirini doesn't support the `/tasks` endpoint; but the comment should describe why just removing it is safe.)

Reply:
I've added comments to the commit message. Will force-push amended commits.